cmd/services.go

```diff
@@ -257,7 +257,6 @@ func restartAllServices(config client.AgentConfig, isServer bool) {
 		err = startService(conn, serviceName)
 		if err != nil {
 			log.Error(err, "Unable to start service", "name", serviceName)
-			panic(err)
```

Comment:
i'm not sure it's a good idea to ignore service start failures. currently, this causes the agent to exit and get restarted by systemd, which allows it to keep retrying to recover from any failures.